Regression test collate.icu.utf8 fails with the following error:

    ERROR:  collations with different collate and ctype values are not supported by ICU
    CONTEXT:  SQL statement "CREATE COLLATION test1 (provider = icu, lc_collate = 'C', lc_ctype = 'C.UTF-8');"
    PL/pgSQL function inline_code_block line 3 at EXECUTE

I don't know how to fix nor what's the cause. Unsetting LC_COLLATE (it's set
to `C` by /etc/profile.d/locale.sh) or setting it to `C.UTF-8` didn't help.

--- a/src/test/regress/parallel_schedule
+++ b/src/test/regress/parallel_schedule
@@ -78,7 +78,7 @@
 # ----------
 # Another group of parallel tests
 # ----------
-test: create_table_like alter_generic alter_operator misc async dbsize misc_functions sysviews tsrf tid tidscan collate.icu.utf8 incremental_sort
+test: create_table_like alter_generic alter_operator misc async dbsize misc_functions sysviews tsrf tid tidscan incremental_sort
 
 # rules cannot run concurrently with any test that creates
 # a view or rule in the public schema
